### ISSUE TITLE
Do we have a dependency on some random depo for JXC?

### ISSUE THREAD

**Issue by yaronyg** [thaliproject/jxcore#50]
I noticed in the logs:

jxtools@0.0.4  (git+https://github.com/ktrzeciaknubisa/jxtools.git#644cf31ea259cb65a97e5c3ed5ea1236dba298a3)

So do we depend in JXC on some random depot in GitHub?

**Comment by enricogior** [comment]
Yes, https://github.com/jxcore/jxc/blob/master/package.json has this dependency:
```
  "dependencies": {
    "jxtools": "git+https://github.com/ktrzeciaknubisa/jxtools.git"
  }
```

**Comment by yaronyg** [comment]
How do we stabilize this dependency? That git hub repo isn't even part of the JXcore repo! The owner is a former employee of Nubisa but still we should probably figure out how to get this code somewhere more stable.

**Comment by enricogior** [comment]
A possible solution is to fork the repo to thaliproject.
